tests/integration/Utils.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace OpenStack\Integration;
+
+use GuzzleHttp\Client;
+use OpenStack\Identity\v2\Api;
+use OpenStack\Identity\v2\Service;
+use OpenStack\Common\Transport\HandlerStack;
+use OpenStack\Common\Transport\Utils as CommonUtils;
+
+class Utils
+{
+    public static function getAuthOptsV3()
+    {
+        return [
+            'authUrl' => getenv('OS_AUTH_URL'),
+            'region'  => getenv('OS_REGION_NAME'),
+            'user'    => [
+                'id'       => getenv('OS_USER_ID'),
+                'password' => getenv('OS_PASSWORD'),
+            ],
+            'scope'   => [
+                'project' => [
+                    'id' => getenv('OS_PROJECT_ID'),
+                ]
+            ]
+        ];
+    }
+
+    public static function getAuthOptsV2()
+    {
+        $httpClient = new Client([
+            'base_uri' => CommonUtils::normalizeUrl(getenv('OS_AUTH_URL')),
+            'handler'  => HandlerStack::create(),
+        ]);
+        $identityService = new Service($httpClient, new Api);
+        return [
+            'authUrl'         => getenv('OS_AUTH_URL'),
+            'region'          => getenv('OS_REGION_NAME'),
+            'username'        => getenv('OS_USERNAME'),
+            'password'        => getenv('OS_PASSWORD'),
+            'tenantName'      => getenv('OS_TENANT_NAME'),
+            'identityService' => $identityService,
```

Comment:
nitpicking: can we have

```php
'identityService' => new Service($httpClient, new Api),
```

since we're not using `$identityService` anywhere else